src/components/resume-templates/modern-corp/publications/modern-corp-publications.scss

```diff
@@ -0,0 +1,47 @@
+.modern-corp-school {
```

Comment:
This file has much more css than needed for its componentThis file has much more css than needed for its component